internal/apiserver/server.go

```diff
@@ -476,9 +476,15 @@ func (as *apiServer) createMuxRouter(ctx context.Context, o orchestrator.Orchest
 				Methods(route.Method)
 		}
 	}
+	r.HandleFunc("/api/v1/namespaces/{ns}/apis/{apiName}/ui", func(rw http.ResponseWriter, req *http.Request) {
+		path := strings.TrimSuffix(req.URL.Path, "/ui") + "/apispec"
+		handler := as.apiWrapper(as.swaggerUIHandler(publicURL + path))
+		handler(rw, req)
+	})
+
 	ws, _ := eifactory.GetPlugin(ctx, "websockets")
 	r.HandleFunc(`/api/swagger{ext:\.yaml|\.json|}`, as.apiWrapper(as.swaggerHandler(routes, apiBaseURL)))
-	r.HandleFunc(`/api`, as.apiWrapper(as.swaggerUIHandler(publicURL)))
+	r.HandleFunc(`/api`, as.apiWrapper(as.swaggerUIHandler(publicURL+"/api/swagger.yaml")))
```

Comment:
Trying to understand this one, maybe you could let me know what the intention is here.

Reply:
I changed `swaggerUIHandler()` so that you have to pass in the full URL to the swagger spec (previously we were passing in a prefix and it was inferring the `/api/swagger.yaml` on the end)